app-pda/libusbmuxd/libusbmuxd-1.0.10.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A client library to multiplex connections from and to iOS devices"
+DESCRIPTION="libusbmuxd is a client library to multiplex connections from and \
+to iOS devices by connecting to a socket provided by a usbmuxd daemon"
+HOMEPAGE="http://www.libimobiledevice.org/"
+COPYRIGHT="2013-2014 Nikias Bassen
+	2009-2014 Martin Szulecki"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/libusbmuxd-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1aa21391265d2284ac3ccb7cf278126d10d354878589905b35e8102104fec9f2"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	libusbmuxd$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libusbmuxd$secondaryArchSuffix = 4.0.0 compat >= 4
+	cmd:iproxy = $portVersion compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libplist$secondaryArchSuffix
+	lib:libplist++$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libusbmuxd${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libusbmuxd$secondaryArchSuffix = 4.0.0 compat >= 4
+	"
+REQUIRES_devel="
+	libxml2$secondaryArchSuffix
+	libplist$secondaryArchSuffix
```

Comment:
Please change REQUIRES_devel to:
```
REQUIRES_devel="
	libusbmuxd$secondaryArchSuffix == $portVersion base
	devel:libxml2$secondaryArchSuffix
	devel:libplist$secondaryArchSuffix
	devel:libplist++$secondaryArchSuffix
	"
```

Hmm, It might be better to remove these 3 devel:lib\* lines from REQUIRES_devel.
The rule is that if a header provided by a devel:lib\* (listed in BUILD_REQUIRES) is \#include'd by the headers of libusbmuxd_devel then we need that devel:lib\* in REQUIRES_devel.
But I think that in most cases we just skip these devel:lib\* lines in REQUIRES_devel because they can sometimes add unncecessary dependencies.